packages/frontend/composables/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@n8n/frontend-composables",
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
+    }
+  },
+  "scripts": {
+    "dev": "vite",
+    "build": "pnpm run typecheck && tsup",
+    "preview": "vite preview",
+    "typecheck": "vue-tsc --noEmit",
+    "test": "vitest run",
+    "test:dev": "vitest",
```

Comment:
In `editor-ui` this is `"test:dev": "vitest --silent=false"`. Should we have the `--silent` here as well?

Reply:
Done. Good catch!